src/buffer.c

```diff
@@ -1,11 +1,12 @@
-#include <stdarg.h>
-#include <string.h>
 #include <assert.h>
-#include <string.h>
+#include <limits.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <stdint.h>
-#include <limits.h>
+#include <string.h>
+#include <string.h>
```

Comment:
This looks like a mistake - two identical lines?

Reply:
A latent mistake, but a mistake nonetheless.  It is from RL2, RL4.